# Object3D with region Array message

std_msgs/Header header

# Array of objects with regions
ObjectRegion3D[] objects
